documentation/documentation.html

```diff
@@ -175,12 +175,12 @@
           <hr />
 
           <div id="lists-container"></div>
-          <hr /> 
+          <hr />
 
-          <!-- <div id="modal-container"></div>
+          <div id="modal-container"></div>
           <hr />
 
-          <div id="navigation-container"></div>
+          <!--  <div id="navigation-container"></div>
```

Comment:
Remove commented code, which you are not using currently

Reply:
I have removed all the commented code !!

Reply:
great